docs/development/how-to-add-a-type-with-bindings.md

```diff
@@ -0,0 +1,20 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+# How to add a type with its python bindings
+
+1. Define it in the `libparsec/crates/types`
+2. Implement (or derive):
```

Comment:
```suggestion
2. Implement (or derive) the needed traits:
```

@FirelightFlagboy is that ok ?

Reply:
I'm OK, if the list below is updated as some traits are only required for certain capabilities